geowebcache/core/src/test/java/org/geowebcache/layer/wms/WMSLayerTest.java

```diff
@@ -423,7 +423,7 @@ public HttpResponse executeRequest(
 
         assertEquals(200, servletResp.getStatus());
         assertEquals("Fake body", servletResp.getContentAsString());
-        assertEquals("image/png", servletResp.getContentType());
+        assertEquals("image/png;charset=UTF-8", servletResp.getContentType());
```

Comment:
Binary formats in general should not have UTF-8 (I already commented on this but the comment is for some reason no longer here... oh well).

Reply:
fixed